assets/components/contribAmounts/contribAmounts.jsx

```diff
@@ -238,17 +239,34 @@ const contribCaptionRadios = {
   ],
 };
 
+function getMonthlyAmount(abTests: Participations, currency: IsoCurrency) {
+  if (abTests.ukRecurringAmountsTest === 'higher' || abTests.usRecurringAmountsTest === 'higher') {
+    return amountRadiosMonthlyHigher[currency];
+  }
+
+  if (abTests.ukRecurringAmountsTest === 'lower' || abTests.usRecurringAmountsTest === 'lower') {
+    return amountRadiosMonthlyLower[currency];
+  }
+
+  if (abTests.ukRecurringAmountsTest === 'wildcard' || abTests.usRecurringAmountsTest === 'wildcard') {
+    return amountRadiosMonthlyWildcard[currency];
+  }
+
+  return amountRadiosMonthlyControl[currency];
+
+}
```

Comment:
Won't this catch people who are in the UK in the US test, and people who are in the US in the UK test?

Reply:
Yes it will...but as it is impossible to be on the UK page and the US page at the same time, I don't think it matters.